extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -52,6 +52,7 @@ attr_lists: {
     name: "disableremoteplayback"
     value: ""
   }
+  attrs: { name: "dock" }
```

Comment:
probably should have `value: ""`

Reply:
It can have a non-empty value.

Reply:
Oh apologies, I didn't see any usage of it being some string in the examples.